packages/react-composites/src/composites/CallWithChatComposite/SidePane.tsx

```diff
@@ -143,6 +177,19 @@ export const EmbeddedPeoplePane = (props: {
   );
 };
 
+const drawerContainerStyles: IStackStyles = {
+  root: {
+    position: 'absolute',
```

Comment:
You should still make sure that this element's siblings have `position: 'relative'` set so that any zIndex from inside those elements won't compete with this one.

(So, you want to `position:relative` for `participantList`).